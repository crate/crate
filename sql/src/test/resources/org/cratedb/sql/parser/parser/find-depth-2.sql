SELECT 1 IN (1, 2, 3)