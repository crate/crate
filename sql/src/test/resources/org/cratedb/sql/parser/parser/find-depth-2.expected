org.cratedb.sql.parser.parser.CursorNode@bf3fd22
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@3f6fc94f
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@44098212

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@4c3a8ea3
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.InListOperatorNode@186a966e
			operator: IN
			methodName: in
			type: null
			leftOperand: 				
				org.cratedb.sql.parser.parser.RowConstructorNode@7dba1c47
				depth: 0
			rightOperandList: 				
				org.cratedb.sql.parser.parser.RowConstructorNode@4596b9cd
				depth: 1
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@2a80656c